src/test/java/uk/gov/digital/ho/hocs/workflow/processes/DCU_MIN.java

```diff
@@ -0,0 +1,164 @@
+package uk.gov.digital.ho.hocs.workflow.processes;
+
+import org.camunda.bpm.engine.test.Deployment;
+import org.camunda.bpm.engine.test.mock.Mocks;
+import org.camunda.bpm.extension.mockito.ProcessExpressions;
+import org.camunda.bpm.extension.process_test_coverage.junit.rules.TestCoverageProcessEngineRule;
+import org.camunda.bpm.extension.process_test_coverage.junit.rules.TestCoverageProcessEngineRuleBuilder;
+import org.camunda.bpm.scenario.ProcessScenario;
+import org.camunda.bpm.scenario.Scenario;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.digital.ho.hocs.workflow.BpmnService;
+import uk.gov.digital.ho.hocs.workflow.util.ExecutionVariableSequence;
+import uk.gov.digital.ho.hocs.workflow.util.CallActivityReturnVariable;
+
+import java.util.Arrays;
+
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+@RunWith(MockitoJUnitRunner.class)
+@Deployment(resources = {
+        "processes/DCU_MIN.bpmn",
+        "processes/DCU_BASE_DATA_INPUT.bpmn",
+        "processes/DCU_MIN_MARKUP.bpmn",
+        "processes/DCU_MIN_INITIAL_DRAFT.bpmn",
+        "processes/DCU_BASE_QA_RESPONSE.bpmn",
+        "processes/DCU_MIN_MINISTER_SIGN_OFF.bpmn",
+        "processes/DCU_MIN_PRIVATE_OFFICE.bpmn",
+        "processes/DCU_BASE_DISPATCH.bpmn",
+        "processes/STAGE.bpmn",
+        "processes/STAGE_WITH_USER.bpmn"})
+public class DCU_MIN {
+
+    public static final String DISPATCH = "CallActivity_1rowgu5";
+    public static final String PO_SIGN_OFF = "CallActivity_0pmeblj";
+    public static final String INITIAL_DRAFT = "CallActivity_1ket68y";
```

Comment:
This is what Will described as "smoke and mirrors" in his comment here:
https://github.com/UKHomeOffice/hocs-workflow/pull/348
I've just been renaming the workflow boxes to be more meaningful.
Do we have a better solution to Will's point?

Reply:
Will's point was that we're testing the workflows directly. We can't get away from that unless we change the design. My opinion is 'we are where we are' and any tests are better than non. I wasn't sure of the knock on effect of renaming them, and what would happen if there was a duplicate name?

Reply:
We are where we are.
(The modeller prevents entering a duplicate.)